dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,12 +1,6 @@
 {% macro athena__drop_relation(relation) -%}
-  {% set rel_type_object = adapter.get_table_type(relation.schema, relation.identifier) %}
-  {%- if rel_type_object is not none %}
-    {% set rel_type = rel_type_object.value %}
-    {%- if rel_type == 'table' or rel_type == 'iceberg_table' %}
```

Comment:
What is important here is that this check has no need because `clean_up_table` get the table location and only if it is not None or an empty string we delete from S3. In our case:
- If the table does not exist it returns None
- If the table is in fact a view, the location is empty
- If the table is really a table, it returns the path to the table